lib/global_registry_bindings/entity/push_entity_methods.rb

```diff
@@ -50,35 +50,35 @@ def create_dependent_entity_in_global_registry # rubocop:disable Metrics/AbcSize
           ensure_parent_entity_has_global_registry_id!
           entity_attributes = {
             global_registry_entity.parent_type => {
-              client_integration_id: global_registry_entity.parent.id,
+              :client_integration_id => global_registry_entity.parent.id,
```

Comment:
This seems like an odd fix. Maybe it's trying to maintain consistency with the rest of the hash?

Reply:
Yes, the warning was:
```
C: 53: 15: [Correctable] Style/HashSyntax: Don't mix styles in the same hash.
```